### PR TITLE
Discord api

### PR DESCRIPTION
**Pull request by priyamishra2903**
Gbu and mentor-mentee functionality added

### CODE REVIEW

File: devsnest/app/controllers/api/v1/mmts_controller.rb [hunk 1]
```diff
@@ -4,16 +4,16 @@ module Api
...
+      # def context
+      #   { user: current_user }
+      # end
 
-      def authorized
-        return head 403 if current_user.normal?
-      end
+      # def authorized
+      #   return head 403 if current_user.normal?
+      # end
```
Comment (vedansh) [inline devsnest/app/controllers/api/v1/mmts_controller.rb:16]: delete if not required

File: devsnest/app/controllers/api/v1/writeups_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          return render_error("User not found") if user.nil? 
+          # return head 404
+
+          params["data"]["attributes"].delete("discord_id")
+          params["data"]["attributes"]["user_id"] = user.id
+        end
+      end
+    end
+  end
+
```
Comment (vedansh) [inline devsnest/app/controllers/api/v1/writeups_controller.rb:21]: fix indenatation

File: devsnest/app/controllers/api/v1/mmts_controller.rb (hunk 1 above)
Comment (priyamishra2903) [reply devsnest/app/controllers/api/v1/mmts_controller.rb]: resolved

File: devsnest/app/controllers/api/v1/writeups_controller.rb (hunk 2 above)
Comment (priyamishra2903) [reply devsnest/app/controllers/api/v1/writeups_controller.rb:20]: resolved

**Pull request merged**